OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -315,7 +315,8 @@ void ConnectionStateChanged(OrderManager om)
 			var startGameButton = lobby.GetOrNull<ButtonWidget>("START_GAME_BUTTON");
 			if (startGameButton != null)
 			{
-				startGameButton.IsDisabled = () => configurationDisabled() || Map.Status != MapStatus.Available ||
+				startGameButton.IsDisabled = () => (configurationDisabled() && (orderManager.LocalClient != null && !orderManager.LocalClient.IsAdmin)) ||
```

Comment:
Most of the checks in `configurationDisabled` are there for technical reasons that shouldn't be ignored (map has bogus rules, admin has the kick dialog open, the start-game order is currently in-flight across the network).  Why do you need this?